modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -0,0 +1,242 @@
+     outputRefs: <12>
+     - elasticsearch
+     - secureforward-offcluster
+   - name: infra-logs
+     inputSource: logs.infra
+     outputRefs:
+     - elasticsearch-insecure
+   - name: audit-logs
+     inputSource: logs.audit
+     outputRefs:
+     - secureforward-offcluster
+----
+<1> Set the annotation to `enabled` to enable log forwarding.
+<2> Add one or more endpoints:
+* Specify the type of output, either `elasticseach` or `forward`.
+* Enter a name for the output.
+* Enter the endpoint, either the server name or FQDN.
+* Optionally, enter the name of the secret required by the endpoint for TLS communication. The secret must exist in the openshift-logging project.
+* Specify insecure: true if the endpoint does not use a secret, resulting in insecure communication.
+<3> Add one or more pipelines:
+* Enter a name for the pipeline
+* Specify the source type, logs.app, logs.infra, or logs.audit.
+* Specify the name of one or outputs configured in the CR.
+
+[NOTE]
+====
+If you set `disableDefaultForwarding: true` you must configure a pipeline and output for all three types of logs, application, infrastructure, and audit. If you do not specify a pipeline and output for a log type, those logs are not stored and will be lost.
+==== 
+
+== Example log forwarding custom resources
+ 
+A typical Log Forwarding configuration would be similar to the following examples.
+
+The following Log Forwarding custom resource sends all logs to a secured external Elasticsearch instance:
+
+.Sample custom resource to forward to an Elasticsearch instance
+
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogForwarding
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  disableDefaultForwarding: true
+  outputs:
+    - name: user-created-es
+      type: elasticsearch
+      endpoint: 'elasticsearch-server.openshift-logging.svc:9200'
+      secret:
+        name: piplinesecret
+  pipelines:
+    - name: app-pipeline
+      inputSource: logs.app
+      outputRefs:
+        - user-created-es
+    - name: infra-pipeline
+      inputSource: logs.infra
+      outputRefs:
+        - user-created-es
+    - name: audit-pipeline
+      inputSource: logs.audit
+      outputRefs:
+        - user-created-es
+----
+
+The following Log Forwarding custom resource sends all logs to a secured Fluentd instance using the Fluentd `out_forward` plug-in.
+
+.Sample custom resource to use the`out_forward` plugin
+
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogForwarding
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  disableDefaultForwarding: true
+  outputs:
+    - name: fluentd-created-by-user
+      type: forward
+      endpoint: 'fluentdserver.openshift-logging.svc:24224'
+      secret:
+        name: fluentdserver
+  pipelines:
+    - name: clo-default-app-pipeline
+      inputType: logs.app
+      outputRefs:
+        - fluentd-created-by-user
+    - name: clo-default-infra-pipeline
```

Comment:
The `clo-default-infra-pipeline` is a default name, please don't use it in the example. If user create a LF instance with this pipeline name, then the pipeline will be dropped.